components/CheckDocumentation/vacuum/XminHorizon.tsx

```diff
@@ -0,0 +1,205 @@
+                <code>{byBackend["xmin"]}</code>. You can find such transaction
+                and its pid by running the following command:
+              </p>
+              <CodeBlock>
+                <SQL
+                  sql={`SELECT pid, datname, usename, state, backend_xmin, backend_xid
+                  FROM pg_stat_activity
+                  WHERE backend_xmin IS NOT NULL OR backend_xid IS NOT NULL
+                  ORDER BY greatest(age(backend_xmin), age(backend_xid)) DESC;`}
+                />
+              </CodeBlock>
+              <p>
+                You can cancel it with{" "}
+                <SQL inline sql={`SELECT pg_cancel_backend('<query_pid>');`} />{" "}
+                or{" "}
+                <SQL
+                  inline
+                  sql={`SELECT pg_terminate_backend('<query_pid>');`}
+                />
+                .
+              </p>
+            </>
+          )}
+          {byReplicationSlot && (
+            <p>
+              The replication slot is holding back the xmin horizon at{" "}
+              <code>{byReplicationSlot["xmin"]}</code>. This indicates that some
+              physical (streaming) replication slot is likely either delayed or
+              down. If the replication slot is no longer used, remove it with{" "}
+              <SQL
+                inline
+                sql={`SELECT pg_drop_replication_slot('<slot_name>');`}
+              />
+              .
+            </p>
+          )}
+          {byReplicationSlotCatalog && (
+            <p>
+              The replication slot is holding back the xmin horizon at{" "}
+              <code>{byReplicationSlotCatalog["xmin"]}</code>, specifically with
+              the system catalogs. This indicates that the replication slot is
+              either delayed, down, or having trouble replicating due to schema
+              mismatch. If the replication slot is no longer used, remove it
+              with{" "}
+              <SQL
+                inline
+                sql={`SELECT pg_drop_replication_slot('<slot_name>');`}
+              />
+              .
+            </p>
+          )}
+          {byStandby && (
+            <>
+              <p>
+                The long running transaction on the standby is holding back the
+                xmin horizon at <code>{byStandby["xmin"]}</code>. You can find
+                the <code>xmin</code> of all standby servers by running the
+                following command:
+              </p>
+              <CodeBlock>
+                <SQL
+                  sql={`SELECT application_name, client_addr, backend_xmin
+                  FROM pg_stat_replication
+                  ORDER BY age(backend_xmin) DESC;`}
+                />
+              </CodeBlock>
+            </>
+          )}
+          {byPreparedXact && (
+            <>
+              <p>
+                The orphaned prepared transaction is holding back the xmin
```

Comment:
```suggestion
                A prepared transaction is holding back the xmin
```
I think above it makes sense to talk about abandoned prepared transactions ("abandoned" probably makes more sense than "orphaned" since they don't necessarily have a parent in any meaningful sense), but here I don't think we want to imply that we have _identified_ this prepared transaction as something that definitely needs to be rolled back. It's possible that users actually want to commit it; we can let them make that call. I think the rest of the guidance can still focus on the likely "roll back" scenario.